docs/manual/introduction/Optimizations.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<base href="../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		<h1>[name]</h1>
+
+		<p class="desc">
+			This article shows how to optimize your three.js scene.
+		</p>
+
+		<h2>Metrics</h2>
+
+		<p>
+			Before starting, we need to define a some metrics, and how to monitor them. There are two important factors
+			that will affect your performances:
+
+			<ul>
+				<li>CPU: affects the number of frames per seconds (FPS) which represents how fluid is your scene</li>
+				<li>Memory: affects the memory usage which can be restricted on low devices like smartphones</li>
+			</ul>
+		</p>
+
+		<p>
+			In order to monitor them you can use [mrdoob/stats.js](https://github.com/mrdoob/stats.js) repository.
+		</p>
+
+		<h2>Buffer Geometries</h2>
+
+        <p>
+            BufferGeometries store information (such as vertex positions, face indices, normals, colors,
+            UVs, and any custom attributes) in [page:BufferAttribute buffers] - that is,
+            [link:https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays typed arrays].
+        </p>
+
+        <p>
+            If your are dealing with static scene, or a huge one and need to reduce memory usage you should rather use
+            BufferGeometries instead of Geometries which are heavier.
+        </p>
+
+        <h2>Share Materials & Geometries</h2>
+
+        <p>
+            If you have multiple times the same object you want to repeat inside your scene. You muse share both material
+            and geometry of that object in order to prevent memory overhead.
+        </p>
+
+        <code>
+var geometry = new THREE.SphereBufferGeometry( 5, 32, 32 );
+var material = new THREE.MeshBasicMaterial( {color: 0xffff00} );
+var sphere;
+
+for (var i = 0; i < 100; i++) {
+    sphere = new THREE.Mesh( geometry, material );
+    sphere.position.set( Math.random() * 100, Math.random() * 100, Math.random() * 100);
+
+    scene.add( sphere );
+}
+        </code>
+
+        <p>
+            The counter-part is that when you modify the geometry or material, you modification will be applied to all
+            instances.
+        </p>
+
+        <p>
+            Note that using [page:Mesh.clone] will share geometry and material.
+        </p>
+
+		<h2>Disable Scene auto update</h2>
+
+        <p>
+            By default [page:Scene.autoUpdate] is set true meaning at each frame you will update the world matrix. This
+            is fine for most of the cases but when you deal with huge scene and especially if
+            [page:Object3D.matrixAutoUpdate] is set true, which is the default case this could have a very bad impact.
+        </p>
```

Comment:
Maybe explain that matrices do need to be computed once, after `position` `scale` and `rotation` changes? Doesn't have to be every frame, but without it, no objects would appear in world space.